exercises/practice/grep/.docs/instructions.append.md

```diff
@@ -0,0 +1,9 @@
+# dummy
+
+## "io" module
+
+The "io" module is part of [the wren-console extended library][wrenc].
```

Comment:
Not quite correct.  The 5 functions listed are part of the extended library support.  `io` is part of wren-console proper (it was already include in wren-cli which we forked from).  I'd probably mention that IO is a built-in module and that (if relevant) a few features of that module are in the extended library.

Reply:
@joshgoebel  Would  it be appropriate to link to https://wren.io/cli/modules/ ?

Reply:
Sure. But because wren-console adds things that's not going to be a fully complete view.  Though that might not matter here.